sdk/core/Azure.Core.TestFramework/src/TestProxy.cs

```diff
@@ -60,6 +60,8 @@ static TestProxy()
 
         private TestProxy(string proxyPath, bool debugMode = false)
         {
+            debugMode = !string.IsNullOrWhiteSpace(Environment.GetEnvironmentVariable("PROXY_MANUAL_START"));
```

Comment:
Should we OR the value rather than overwriting it?
```suggestion
            debugMode |= !string.IsNullOrWhiteSpace(Environment.GetEnvironmentVariable("PROXY_MANUAL_START"));
```

Reply:
Yeah absolutely. Wouldn't want folks to have their testclass setting ignored.